cmd/ttn-lw-cli/commands/flags.go

```diff
@@ -71,10 +71,12 @@ func mergeAttributes(attributes map[string]string, flagSet *pflag.FlagSet) map[s
 	}
 	for _, kv := range kv {
 		kv := strings.SplitN(kv, "=", 2)
-		if kv[1] == "" {
-			delete(out, kv[0])
-		} else {
-			out[kv[0]] = kv[1]
+		if len(kv) == 2 {
```

Comment:
Early exit?

Reply:
It needs to still range though valid values, so this will just continue to the next item.

Reply:
I mean there's a keyword for continue; `continue`

Reply:
lol. But this does exactly that without using `continue`.